app/assets/javascripts/deploys.js

```diff
@@ -65,7 +65,7 @@ function update_tasks(progress_data) {
     progress_text = task_data.progress + "% - " + task_data.text;
     $span_task_id.html(progress_text);
 
-    if (!task_data.success) {
+    if (!task_data.success || (task_data.progress < 100 && finished)) {
```

Comment:
super minor`nit` substitute magic number 100 with constant `const BAR_PERCENTAGE_COMPLETE` or something related

but it still ok even